superset/assets/src/CRUD/Fieldset.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'react-bootstrap';
+
+import { recurseReactClone } from './utils';
+import Field from './Field';
+
+const propTypes = {
```

Comment:
all non-required props should have a default

Reply:
we should get our linter to enforce this!

Reply:
turned off `react/require-default-props` and got 222 problems.  Opened an issue https://github.com/apache/incubator-superset/issues/5625